package.json

```diff
@@ -163,6 +165,7 @@
     "pretest": "npm run build",
     "pretest:coverage": "npm run build",
     "posttest": "rimraf ./test.asyncapi-cli",
+    "postinstall": "node ./scripts/enableAutoComplete.js",
```

Comment:
Does this script runs after packaging as well?
Also what about docker image. Does this still work there?
Same question for chocolatey.